api/desecapi/serializers.py

```diff
@@ -22,14 +23,23 @@
 
 class CaptchaSerializer(serializers.ModelSerializer):
     challenge = serializers.SerializerMethodField()
+    kind = serializers.ChoiceField(
+        choices=['image', 'audio'],  # TODO this list could be automatically generated
```

Comment:
`Captcha.Kind` inherits from `enum.Enum`. You should therefore be able to do:

```
choices=list(map(lambda t: t.value, Captcha.Kind))  # or list comprehension
```

However, [the `choices` argument also accepts "a list of `(key, display_name)` tuples"](https://www.django-rest-framework.org/api-guide/fields/#choicefield). It may be sufficient to do `choices=Captcha.Kind` or at least `choices=list(Captcha.Kind)`.

Reply:
seems like it is sufficient to not declare this at all :-)